### ISSUE TITLE
What to do with the small module description articles?

### ISSUE THREAD

**Issue by eriksiegel** [eXist-db/documentation#203]
The documentation set contains currently 2 articles that describe function modules (xmldb, request+session)  that claim to be complete but are not by far. The information in there is useful, no doubt, but (if you claim to describe a module) not enough.

Also: Of all the function modules, only these two get this kind of incomplete coverage. Unfair... :)

So: The documentation is inconsistent with regards to describing modules

What to do with this?
1. Create articles for *all* function modules. Whew, lot of work and probably unnecessary
2. Remove these specific articles?
3. Review the articles and see what needs to be kept
4. Keep things as is

I'm in favor of 3. IMHO the http-request-session article is so flimsy it can go. The xmldb article can get a makeover and retargeted to be about the use of the various collection() functions.

Opinions?

PS: There was such an article about the util module as well. This contained next to nothing so removed it.

*   exist-db version: `4.0.0`
*   documentation version: `4.0.0`

**Comment by duncdrum** [comment]
yes the question of what to do with documentation for the function modules is tricky. What this boils down to, imv, is the question of how the function documentation app and the documentation app are supposed to coexist. 

@dizzzz any thoughts on this?

**Comment by duncdrum** [comment]
btw @line-o about function docs and documentation apps

**Comment by duncdrum** [comment]
both `http-request` and `xmldb` should be aware of this https://github.com/eXist-db/exist-markdown/issues/4

**Comment by duncdrum** [comment]
@joewiz this is the place for discussing where to put the docs for repos/modules/ etc
